### PR TITLE
Remove BrandedComponent

### PR DESCRIPTION
**Pull request by regiskuckaertz**

### CODE REVIEW

File: common/app/templates/inlineJS/blocking/config.scala.js [hunk 1]
```diff
@@ -27,13 +27,6 @@ window.guardian = {
         },
         "tests": { @JavaScript(mvt.ActiveTests.getJavascriptConfig) },
         "modules": { },
-        "images": {
-            "commercial": {
-                "brandedComponentJobs": "@Static("images/commercial/branded-component-jobs.png")",
```
Comment (Calanthe) [inline common/app/templates/inlineJS/blocking/config.scala.js]: Are you going to remove those images from the codebase as well?

Comment (regiskuckaertz) [inline common/app/templates/inlineJS/blocking/config.scala.js]: @Calanthe you're right, sorry I thought all the pictures were in src

Comment (regiskuckaertz) [inline common/app/templates/inlineJS/blocking/config.scala.js]: done

**Comment by kenlim** [comment]
@regiskuckaertz has confirmed that Branded Components are an obsolete component that isn't in use anymore. 

In which case, :+1:

**Pull request merged**